src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { applyMiddleware, createStore, combineReducers } from 'redux';
+import logger from 'redux-logger';
```

Comment:
- [ ] Please Imports configureStore  from @reduxjs/toolkit instead of createStore
![image](https://user-images.githubusercontent.com/74744787/185762052-bcb70de8-7592-427c-a52a-b2e3472755d8.png)